readthedocs/doc_builder/python_environments.py

```diff
@@ -0,0 +1,244 @@
+            'pillow==2.6.1',
+            'readthedocs-sphinx-ext==0.5.4',
+            'sphinx-rtd-theme==0.1.9',
+            'alabaster>=0.7,<0.8,!=0.7.5',
+            'recommonmark==0.1.1',
+        ]
+
+        cmd = [
+            'python',
+            self.venv_bin(version=self.version.slug, filename='pip'),
+            'install',
+            '--use-wheel',
+            '-U',
+            '--cache-dir',
+            self.project.pip_cache_path,
+        ]
+        if self.config.use_system_site_packages:
+            # Other code expects sphinx-build to be installed inside the
+            # virtualenv.  Using the -I option makes sure it gets installed
+            # even if it is already installed system-wide (and
+            # --system-site-packages is used)
+            cmd.append('-I')
+        cmd.extend(requirements)
+        self.build_env.run(
+            *cmd,
+            bin_path=self.venv_bin(version=self.version.slug)
+        )
+
+    def install_user_requirements(self):
+        requirements_file_path = self.config.requirements_file
+        if not requirements_file_path:
+            builder_class = get_builder_class(self.project.documentation_type)
+            docs_dir = (builder_class(build_env=self.build_env, python_env=self)
+                        .docs_dir())
+            for path in [docs_dir, '']:
+                for req_file in ['pip_requirements.txt', 'requirements.txt']:
+                    test_path = os.path.join(self.checkout_path, path, req_file)
+                    if os.path.exists(test_path):
+                        requirements_file_path = test_path
+                        break
+
+        if requirements_file_path:
+            self.build_env.run(
+                'python',
+                self.venv_bin(version=self.version.slug, filename='pip'),
+                'install',
+                '--exists-action=w',
+                '--cache-dir',
+                self.project.pip_cache_path,
+                '-r{0}'.format(requirements_file_path),
+                cwd=self.checkout_path,
+                bin_path=self.venv_bin(version=self.version.slug)
+            )
+
+
+class Conda(PythonEnvironment):
+
+    def venv_path(self, version=LATEST):
+        return os.path.join(self.project.doc_path, 'conda', version)
+
+    def setup_base(self):
+        conda_env_path = os.path.join(self.project.doc_path, 'conda')
+        version_path = os.path.join(conda_env_path, self.version.slug)
+
+        if os.path.exists(version_path):
+            # Re-create conda directory each time to keep fresh state
+            self._log('Removing existing conda directory')
+            shutil.rmtree(version_path)
+        self.build_env.run(
+            'conda',
+            'create',
+            '--yes',
+            '--name',
+            self.version.slug,
+            'python={python_version}'.format(python_version=self.config.python_version),
+            environment={'CONDA_ENVS_PATH': conda_env_path}
```

Comment:
Ah, ok. Thanks for clarifying. Have never actually overridden this one.

So, are the environments somewhere different from the base install?

In any event, having this variable will probably make this change even easier.